src/main/java/com/exactpro/th2/codec/csv/CsvCodec.java

```diff
@@ -155,6 +156,27 @@ public void handler(String consumerTag, RawMessageBatch message) {
         }
     }
 
+    private void fillCsvHeaderMessage(Message.Builder headerMsg, String[] strings) {
+        ListValue.Builder headerValues = ListValue.newBuilder();
+        for (String string : strings) {
+            headerValues.addValues(Value.newBuilder().setSimpleValue(string).build());
+        }
+
+        headerMsg.putFields(HEADER_FIELD_NAME, Value.newBuilder()
+                .setListValue(headerValues)
+                .build());
```

Comment:
`Value.newBuilder().setListValue(headerValues).build()` can also be replaced with `ValueUtilsKt.toValue(headerValues)`